tasks/id.go

```diff
@@ -0,0 +1,18 @@
+package tasks
+
+// Marathon Task Id
+// Usually in the form of AppId.uuid with '/' replaced with '_'
+type Id string
+
+// Marathon Application Id (aka PathId)
+// Usually in the form of /rootGroup/subGroup/subSubGroup/appId
+// allowed characters: lowercase letters, digits, hyphens, slash
+type AppId string
+
+func (id AppId) String() string {
+	return string(id)
+}
+
+func (id Id) String() string {
```

Comment:
moving it closer to the type definition (just below it) would make it nicer to read. :heart: